### PR TITLE
Added UI for anointing amulets

### PR DESCRIPTION
**Pull request by dclamage**
Includes the following UI:
 - Button to anoint when editing an amulet
 - List of notables with tooltips showing what the notable does and its recipe
 - Sorting by the standard power options

### CODE REVIEW

File: Classes/CalcsTab.lua [hunk 1]
```diff
@@ -500,8 +500,8 @@ function CalcsTabClass:CalculatePowerStat(selection, original, modified)
 		original = original.Minion
 		modified = modified.Minion
 	end
-	originalValue = original[selection.stat] or 0
-	modifiedValue = modified[selection.stat] or 0
+	local originalValue = original[selection.stat] or 0
```
Comment (dclamage) [inline Classes/CalcsTab.lua:503]: I noticed and fixed this mistake while working on this PR. I can revert it if the change is inappropriate.

**Comment by dclamage** [comment]
Updated to support anointing non-amulet blight uniques, and double enchant/anoint for blight unique helmets.

File: Classes/CalcsTab.lua (hunk 1 above)
Comment (ppoelzl) [reply Classes/CalcsTab.lua:503]: Could you please refactor this out into a separate PR? The rest is looking great!

**Comment by dclamage** [comment]
Fixed imported items that can be anointed. The text "This item can be anointed by Cassia" was in the flavour text.

**Comment by dclamage** [comment]
Removed an unnecessary change and moved it to a different PR

**Comment by dclamage** [comment]
Rearranged the UI a bit based on feedback.

**Pull request merged**